src/test/java/TestRunner.java

```diff
@@ -106,10 +120,9 @@ public void testThatComparingLinkCreationFeatureGeneratesLinkAndItWorksProperly(
         Assert.assertEquals(newComparingPage.amountOfComparingProducts(), amountOfComparingProducts,
                 String.format("Amount of comparing products = %s doesn't meet expected amount %s.\n",
                         comparingPage.amountOfComparingProducts(), ++amountOfComparingProducts));
-
     }
 
-    @Test(enabled = false)
+    @Test(enabled = true)
     public void testFavoriteShopsFeature() throws InterruptedException {
```

Comment:
Please, do not push to remote repository not implemented/ready code. And not useful comments.